sw/host/opentitanlib/src/otp/alert_handler.rs

```diff
@@ -258,10 +258,11 @@ impl AlertRegs {
             AlertClass::X => bail!("Bad class"),
         };
 
-        let mut reg = 0 as u32;
+        let mut reg = 0_u32;
 
-        reg |= (0 & ALERT_HANDLER_CLASSA_CTRL_SHADOWED_MAP_E0_MASK)
-            << ALERT_HANDLER_CLASSA_CTRL_SHADOWED_MAP_E0_OFFSET;
+        // TODO(lowRISC/opentitan#15443): Fix this lint (clippy::erasing_op):
+        //reg |= (0 & ALERT_HANDLER_CLASSA_CTRL_SHADOWED_MAP_E0_MASK)
+        //    << ALERT_HANDLER_CLASSA_CTRL_SHADOWED_MAP_E0_OFFSET;
```

Comment:
@jon-flatley please correct me if I'm wrong but this line is indeed NOP and exists for the sake of completeness, right?

![image](https://user-images.githubusercontent.com/57949550/196763330-ad355308-f1b5-4956-abc6-983ea37810f5.png)

I also confirmed that this part matches the hard-coded logic in `sw/device/silicon_creator/lib/drivers/alert.c:alert_class_configure()`